moped-editor/src/views/projects/projectView/DeleteConfirmationModal.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import DeleteIcon from "@material-ui/icons/DeleteOutlined";
+
+const useStyles = makeStyles((theme) => ({
+  deleteButton: {
+    color: theme.palette.error.main,
+  },
+}));
+
+const DeleteConfirmationModal = ({
+  type,
+  submitDelete,
+  deleteConfirmationOpen,
+  setDeleteConfirmationOpen,
+  children
+}) => {
+  const classes = useStyles();
+
+  const handleDeleteClose = () => {
+    setDeleteConfirmationOpen(false);
+  };
+
+  return (
+    <span>
+      {children}
+      <Dialog
+        open={deleteConfirmationOpen}
+        onClose={handleDeleteClose}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            {`Are you sure you want to delete this ${type}?`}
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <Button
+            variant="outlined"
+            className={classes.deleteButton}
+            startIcon={<DeleteIcon />}
+            onClick={() => {
+              submitDelete();
+              handleDeleteClose();
+            }}
+          >
+            <span>Delete</span>
+          </Button>
+          <Button
+            variant="outlined"
+            color="primary"
+            onClick={handleDeleteClose}
+            autoFocus
+          >
```

Comment:
i've changed my mind—would mind putting the "Delete" button to the right of the "Cancel" button? That will match the "Delete project" modal, which feels right to me.

would you also switch the "Delete" button's variant to `contained`, and leave the "Cancel" button without a variant?